app/src/main/java/com/cornellappdev/android/eatery/network/JsonUtilities.java

```diff
@@ -75,4 +79,41 @@ public static String loadJSONFromAsset(Context context, String fileName) {
 		}
 		return eateryList;
 	}
+
+    public static ArrayList<EateryBaseModel> parseEateries(List<AllEateriesQuery.Eatery> eateries, Context mainContext) {
+        ArrayList<EateryBaseModel> eateryList = new ArrayList<EateryBaseModel>();
+        JSONObject hardCodedData = null;
+        try {
+            hardCodedData = new JSONObject(loadJSONFromAsset(mainContext, "externalEateries.json"));
+            JSONArray hardCodedEateries = hardCodedData.getJSONArray("eateries");
+            for (int i = 0; i < hardCodedEateries.length(); i++) {
```

Comment:
can we not iterate using something like `for (a : b)`?

Reply:
I'll see if I can change this, this was directly taken from the previous logic.

Reply:
Nope, JSONObject doesn't support iterators as far as I can tell.